db/migrate/20211102180650_remove_message_bus.rb

```diff
@@ -0,0 +1,30 @@
+class RemoveMessageBus < ActiveRecord::Migration[5.2]
+  def up
+    drop_table :message_bus
+  end
+
+  def down
+    ActiveRecord::Base.connection.execute """
```

Comment:
I'm not sure we need this `#down` method at all. It can't restore any deleted data anyway. Also we can't use `MessageBus` methods to recreate the table because that class is not present anymore. So I just took a dump of the DDL. Should I remove it?